cvss/cvss3.py

```diff
@@ -258,17 +263,34 @@ def compute_modified_isc_base(self):
                                       ),
                                      D('0.915'))
 
-    def compute_modified_isc(self):
+    def compute_modified_isc_30(self):
         """
+        This is CVSS:3.0 version
+
         If Modified Scope Unchanged    6.42 x [ISCModified]
-        If Modified Scope Changed    7.52 x [ISCModified-0.029] - 3.25 x [ISCModified-0.02]^15
+        If Modified Scope Changed      7.52 x [ISCModified-0.029] - 3.25 x [ISCModified-0.02]^15
         """
         if self.modified_scope == 'U':
             self.modified_isc = D('6.42') * self.modified_isc_base
         else:  # Modified scope has always value, if not defined then matches Scope
             self.modified_isc = (D('7.52') * (self.modified_isc_base - D('0.029')) -
                                  D('3.25') * (self.modified_isc_base - D('0.02')) ** D('15'))
 
+    def compute_modified_isc(self):
+        """
+        This is CVSS:3.1 version
+
+        If Modified Scope Unchanged    6.42 x [ISCModified]
+        If Modified Scope Changed      7.52 x (ISCModified - 0.029) - 3.25 x
+                                       (ISCModified x 0.9731 - 0.02)^13
+        """
+        if self.modified_scope == 'U':
+            self.modified_isc = D('6.42') * self.modified_isc_base
+        else:  # Modified scope has always value, if not defined then matches Scope
+            self.modified_isc = (D('7.52') * (self.modified_isc_base - D('0.029')) -
+                                 D('3.25') * (self.modified_isc_base * D('0.9731') - D('0.02'))
+                                 ** D('13'))
```

Comment:
Since the only differences in the formulas are the constants being used, you could keep one formula and parameterize the differences in a look-up table?  What you have is very clear.  Maybe something to consider if there is a 3.2...

Reply:
Good idea. I will leave it for now but as you said, something to keep on mind if 3.2 ever comes.